config/sidekiq.yml

```diff
@@ -3,3 +3,7 @@
 :queues:
   - [default, 1]
   - [mailers, 2]
```

Comment:
Should we remove these 2 queues then? 🤔

Reply:
Yeah we can remove it as it's useless now 👍

Reply:
removed in https://github.com/nimblehq/rails-templates/pull/458/commits/a8868a7757acc526346bae0d11e857a8449d02cf